tests/nat/eval/rag_evaluator/test_rag_evaluate.py

```
@@ -199,9 +209,11 @@ def test_ragas_to_eval_output_nan_handling(rag_evaluator, rag_eval_input, metric
         # Average score should match the expected value (with small tolerance for float ops)
         assert round(eval_output.average_score, 4) == round(expected_avg, 4)
 
-        # Each individual item score should match the expected coercion results
+        # Each individual item score should preserve NaN/None values
         actual_item_scores = [item.score for item in eval_output.eval_output_items]
-        assert actual_item_scores == expected_item_scores
+        assert len(actual_item_scores) == len(expected_item_scores)
+        for actual, expected in zip(actual_item_scores, expected_item_scores):
```

Ruff (0.14.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)